autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicOptionRow.tsx

```diff
@@ -23,7 +23,7 @@ const BasicOptionRow: React.FC<BasicOptionRowProps> = ({ id, value }) => {
     <tr>
       <td>
         <Typography variant="body1" style={{ paddingRight: 8 }} id={`${value}-${id}`}>
-          {`${value.slice(0, 1).toUpperCase().concat(value.slice(1))}:`}
+          {(value === 'asynchronous' ? 'No Meeting Times:' : `${value.slice(0, 1).toUpperCase().concat(value.slice(1))}:`)}
```

Comment:
Honestly I think a better alternative would be to add like `text` prop (might be able to find a better name, maybe `name`?) that you could pass in. That way it would be like `<BasicOptionRow id=0 value='asynchronous' name='No Meeting Times' />`, and honors would be `<BasicOptionRow id=0 value='honors' name='Honors' />`, etc. And you'd replace this line with `{name}`

Reply:
done